Week2/wadeea.js

```diff
@@ -0,0 +1,246 @@
+// 1. Arabic - English - Dutch - Italian -Spanish - French.
+console.log(
+  `مرحبا أيها العالم
+  Hello, World!
+  Hallo, wereld!
+  Ciao, mondo!
+  Hola, mundo!
+  Bonjour, monde`,
+);
+
+// 2. The error was because of using the same quotes as the apostrophe in 'I'm'.
+console.log(`I'm awesome`);
```

Comment:
That's correct. You can also use double quotes like: `console.log("I'm awesome");` or use escape characters i.e backslash `\` like: `console.log('I\'m awesome');`. This is just for your information.

Reply:
Thanks. I'll consider this one also.